instrumentation/jdbc/javaagent/src/test/groovy/JdbcInstrumentationTest.groovy

```diff
@@ -495,15 +495,19 @@ class JdbcInstrumentationTest extends AgentInstrumentationSpecification {
       connection = driver.connect(jdbcUrl, null)
     }
 
-    def (Statement statement, ResultSet rs) = runWithSpan("parent") {
+    // TODO: def (Statement statement, ResultSet rs) fails to compile, switch back when this is fixed in spock
```

Comment:
I believe this bug only manifests when test has a `cleanup` block

Reply:
🤯